libraries/Ethernet/src/utility/ethernetif.c

```diff
@@ -128,9 +128,11 @@ void HAL_ETH_MspInit(ETH_HandleTypeDef *heth)
     }
   }
 
+#ifdef ETH_INPUT_USE_IT

```

Comment:
Ok thanks. This is I think an explanation that is needed in the commit message, or in comments where this switch is activated.